package.json

```
@@ -27,21 +28,285 @@
+      "import": "./editors/dropdownType/index.mjs",
+      "require": "./editors/dropdownType/index.js"
+    },
+    "./cellTypes/handsontableType": {
+      "import": "./editors/handsontableType/index.mjs",
+      "require": "./editors/handsontableType/index.js"
+    },
+    "./cellTypes/numericType": {
+      "import": "./editors/numericType/index.mjs",
+      "require": "./editors/numericType/index.js"
+    },
+    "./cellTypes/passwordType": {
+      "import": "./editors/passwordType/index.mjs",
+      "require": "./editors/passwordType/index.js"
+    },
+    "./cellTypes/textType": {
+      "import": "./editors/textType/index.mjs",
+      "require": "./editors/textType/index.js"
+    },
+    "./cellTypes/timeType": {
+      "import": "./editors/timeType/index.mjs",
+      "require": "./editors/timeType/index.js"
+    },
     "./editors": {
       "import": "./editors/index.mjs",
       "require": "./editors/index.js"
     },
+    "./editors/autocompleteEditor": {
+      "import": "./editors/autocompleteEditor/index.mjs",
+      "require": "./editors/autocompleteEditor/index.js"
+    },
+    "./editors/baseEditor": {
+      "import": "./editors/baseEditor/index.mjs",
+      "require": "./editors/baseEditor/index.js"
+    },
+    "./editors/checkboxEditor": {
+      "import": "./editors/checkboxEditor/index.mjs",
+      "require": "./editors/checkboxEditor/index.js"
+    },
+    "./editors/dateEditor": {
+      "import": "./editors/dateEditor/index.mjs",
+      "require": "./editors/dateEditor/index.js"
+    },
+    "./editors/dropdownEditor": {
+      "import": "./editors/dropdownEditor/index.mjs",
+      "require": "./editors/dropdownEditor/index.js"
+    },
+    "./editors/handsontableEditor": {
+      "import": "./editors/handsontableEditor/index.mjs",
+      "require": "./editors/handsontableEditor/index.js"
+    },
+    "./editors/numericEditor": {
+      "import": "./editors/numericEditor/index.mjs",
+      "require": "./editors/numericEditor/index.js"
+    },
+    "./editors/passwordEditor": {
+      "import": "./editors/passwordEditor/index.mjs",
+      "require": "./editors/passwordEditor/index.js"
+    },
+    "./editors/selectEditor": {
+      "import": "./editors/selectEditor/index.mjs",
+      "require": "./editors/selectEditor/index.js"
+    },
+    "./editors/textEditor": {
+      "import": "./editors/textEditor/index.mjs",
+      "require": "./editors/textEditor/index.js"
+    },
     "./plugins": {
       "import": "./plugins/index.mjs",
       "require": "./plugins/index.js"
     },
+    "./plugins/autoColumnSize": {
+      "import": "./plugins/autoColumnSize/index.mjs",
+      "require": "./plugins/autoColumnSize/index.js"
+    },
+    "./plugins/autofill": {
+      "import": "./plugins/autofill/index.mjs",
+      "require": "./plugins/autofill/index.js"
+    },
+    "./plugins/autoRowSize": {
+      "import": "./plugins/autoRowSize/index.mjs",
+      "require": "./plugins/autoRowSize/index.js"
+    },
```

Is it possible to create a wildcard here instead listing all plugins?

```suggestion
    "./plugins/*": {
      "import": "./plugins/*/index.mjs",
      "require": "./plugins/*/index.js"
    },
```

---

Thanks for noticing! I tried this syntax a couple of days ago but without success. However now it seems to work 🤔  I need to check that again especially with older versions of NodeJS. If I do not find any issues I will update this exports config.

---

I've tested the wildcard on the latest Node and webpack5. As I assumed this syntax doesn't work yet.

---

Ok, we will reevaluate this in the future. Thanks for checking